manifests/tftp.pp

```diff
@@ -29,4 +29,75 @@
       root => $foreman_proxy::tftp_root,
     }
   }
+
+  case $::osfamily {
+    'RedHat': {
+      case $::operatingsystem {
+        'Fedora': {
+          if versioncmp($::operatingsystemrelease, '19') >= 0 or $::operatingsystemrelease == 'Rawhide' {
```

Comment:
I don't know if operatingsystemrelease can include Rawhide (never used it), but maybe it's more future proof to check if <= 6 then use redhat_old.